lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -32,8 +29,9 @@
 }
 
 .FilePickerApp__document-source-buttons {
-  align-items: stretch;
+  @include layout.column($align: 'flex-start');
+  // Override layout.column's display from "flex" to "inline-flex" so the buttons don't
+  // take the full width
   display: inline-flex;
```

Comment:
This ended up being one way to keep the buttons from going full width. I had trouble if the display was set to "flex", I do wonder if there is an even simpler or better approach?

Reply:
My gut hunch is that there probably is a more straightforward way, but I am not certain: I'd have to get in and poke around.

One suggestion: perhaps don't try to use any mixins or utilities and just see what CSS is needed to get the job done. A columnar flex layout does seem plausible...But the `inline-flex` display gives me pause.

Reply:
After some careful consideration, I think what is here currently seems reasonable. We have a flex container set to vertical, but we don't want to take the full width, so we use inline-flex. Button items then flow and fill up the space that the longest one takes.